src/midi-router.js

```diff
@@ -9,7 +9,7 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking");
+    flock = fluid.require("flocking"); // eslint-disable-line no-unused-vars
```

Comment:
If it really is unused, why not avoid the variable assignment and change it to

<code>fluid.require("flocking");</code>

?